ibis/backends/tests/test_temporal.py

```diff
@@ -792,6 +800,12 @@ def convert_to_offset(x):
                     raises=com.OperationNotDefinedError,
                     reason="Some wonkiness in sqlglot generation.",
                 ),
+                pytest.mark.notyet(
+                    ["pyspark"],
+                    condition=IS_SPARK_REMOTE,
+                    raises=PySparkConnectGrpcException,
+                    reason="arrow conversion breaks",
```

Comment:
non-duration intervals in arrow are extremely hard to use, i'm surprised only these two test cases are failing here.